cli/cli.go

```diff
@@ -391,23 +395,53 @@ documentation for detail information about writing tests.`,
 	}
 }
 
+func migrationSubcommandWithLock(action func(*cli.Context)) func(*cli.Context) {
+	return func(context *cli.Context) {
+		configFile := context.String("config-file")
+		if migration.LoadConfig(configFile) != nil {
+			return
+		}
+
+		useEtcd := context.Bool(useEtcdDuringMigrationsFlag)
+		if !useEtcd {
+			log.Warning("Do not run multiple processes in parallel")
```

Comment:
I think this message make people misunderstand that they have already executed another migration process.